components/gems/Gemfile

```diff
@@ -101,7 +101,7 @@ group(:omnibus_package) do
   gem "chef-sugar"
```

Comment:
We should consider removing this since we ship this in chef 15.X more or less

Reply:
to clarify what @tas50, we've put functionality in chef-utils, which ships in core chef, that is mostly equivalent to chef sugar starting around 15.5+